contrib/experimental/great_expectations_experimental/expectations/expect_column_discrete_entropy_to_be_between.py

```diff
@@ -302,10 +293,10 @@ def validate_configuration(
     @render_evaluation_parameter_string
     def _prescriptive_renderer(
         cls,
-        configuration=None,
-        result=None,
-        language=None,
-        runtime_configuration=None,
+        configuration: Optional[ExpectationConfiguration] = None,
+        result: Optional[ExpectationValidationResult] = None,
+        language: Optional[str] = None,
+        runtime_configuration: Optional[dict] = None,
         **kwargs,
```

Comment:
Do we know anything about kwargs and the return? Same question throughout.

Reply:
`**kwargs` can possibly be removed, but I'm planning to look into that in a subsequent PR. The return type annotation is going to be a much more manual process than this find/replace because the renderers can return many different types. I will look into how to accomplish that later as well.